content/en/blog/_posts/2022-11-27-Vulnerable-Microservices/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: blog
+title: "Vulnerable Microservices"
```

Comment:
This looks like we're reporting a vulnerability in Kubernetes, or might do. Please tweak the title to avoid that.